### ISSUE TITLE
Training on 1 Gpu

### ISSUE THREAD

**Issue by uurcelikk** [HUANGLIZI/LViT#32]
Hello, What kind of optimisation can I do to train on 1 gpu?

**Comment by HUANGLIZI** [comment]
You should reduce the batch size and also the learning rate to get optimal performance on 1 GPU.
